qa/suites/rbd/persistent-writeback-cache/pool/rwl.yaml

```diff
@@ -0,0 +1,10 @@
+tasks:
+- install:
+- ceph:
+    conf:
+      client:
+        rbd_persistent_cache_mode: rwl
+        rbd_rwl_path: /tmp
```

Comment:
I suspect that this path will be a `tmpfs` and will fail since it doesn't support direct IO. Might need to use a directory under `~ubuntu/cephtest` and make sure you delete it at the end of the test run.

Reply:
DONE.